### PR TITLE
Tas 52 add forgot password link and functionality

### PR DESCRIPTION
**Pull request by psavarmattas**
- Updated: Navigation graph for better understanding of the application structure.
- Added: Forget Password for logged in users so that they can directly change their password.
- Added: Forget Password for not logged in users so that they can receive a link in their inbox for resetting the password.
- Removed: SignupFragment.java copy from the project as it was no longer needed.

### CODE REVIEW

File: local.properties
Comment (kajal1106) [inline local.properties:1]: We are not supposed to commit this file as this has your local sdk path

File: app/src/main/java/com/taskmuse/app/ui/fragment/SignupFragment.java
[the remark this answers is not in the document]
Comment (psavarmattas) [reply app/src/main/java/com/taskmuse/app/ui/fragment/SignupFragment.java:1]: As this file is moved under the authentication package. This is a copy file.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -7,10 +7,11 @@
     <string name="login_page_image_text">Login Page</string>
     <string name="forgot_password_login_view">Forgot Password?</string>
     <string name="new_here_register_button_on_login_view">New Here? Register!</string>
-    <string name="alreadyUser_button_on_register_view">New Here? Register!</string>
+    <string name="alreadyUser_button_on_register_view">New Here? Login!</string>
```
Comment (Agata-Duda) [inline app/src/main/res/values/strings.xml:10]: If the user already exists then it doesn't make sense to say "New Here? Login", instead this should be "Already a member? Login" as per UX designs

**Review by Agata-Duda — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (psavarmattas) [reply app/src/main/res/values/strings.xml]: Updated!

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,11 +7,11 @@
     <string name="login_page_image_text">Login Page</string>
     <string name="forgot_password_login_view">Forgot Password?</string>
     <string name="new_here_register_button_on_login_view">New Here? Register!</string>
-    <string name="alreadyUser_button_on_register_view">New Here? Login!</string>
+    <string name="alreadyUser_button_on_register_view">lready a member? Login!</string>
```
Comment (Agata-Duda) [inline app/src/main/res/values/strings.xml:10]: typo

**Review by Agata-Duda — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (psavarmattas) [reply app/src/main/res/values/strings.xml]: My Bad. Done!

**Review by kajal1106 — APPROVED** [review]

**Review by kajal1106 — CHANGES_REQUESTED** [review]
Typo error

**Comment by psavarmattas** [comment]
@kajal1106 could you comment where the type error is?

**Comment by kajal1106** [comment]
> @kajal1106 could you comment where the type error is?

It's fixed! LGTM!

**Review by kajal1106 — APPROVED** [review]

**Pull request merged**